frontend/src/components/pages/RegistrantExperience/RegistrationSteps/index.tsx

```diff
@@ -363,7 +378,7 @@ const RegistrationSteps = ({
     } else {
       // Append the optional clauses to all the campers before registering the campers
       appendOptionalClausesToCampers(waiverInterface.optionalClauses);
-      registerCampers(campers, edlpChoices);
+      registerCampers(campers, edlpSelections);
```

Comment:
can reviewer please check whether the `appendOptionalClausesToCampers` method is redundant to the `mapToCreateCamperDTO` method?

Reply:
I took a look through them - both are doing the same thing - I think it would still be good to keep the `appendOptionalClausesToCampers` method in case payments fail or something goes wrong with creating the campers. 

How I'm viewing it is `appendOptionalClausesToCampers` would update the fronted state of register campers whereas `mapToCreateCamperDTO` would populate them in our database. 

If we were to remove something I would say we could remove the optionClases map in `mapToCreateCamperDTO` since it should already be stored on camper and thus applied when we spread it